### DISCUSSION TITLE
Does a map like map[string]string work? , because I have this situation:

### DISCUSSION THREAD

**Discussion by Tlepkali** [caarlos0/env#283] [Q&A] [answered]
Does a map like map[string]string work? , because I have this situation:
.env file:
MAP=some-sting:path.json,another-string:path_to_another.json

.go file:
Configs map[string]string `env:"MAP"`

Error:
parsing config failed: env: no parser found for field "Configs" of type "map[string]string"

**Comment by caarlos0** [comment, marked as answer]
as per documentation, yes, it works.

make sure you're using the latest version.
